RELEASE.md

```diff
@@ -27,27 +27,47 @@ work. Here's one example workflow (assuming of course you have rights to
 directly push to the upstream repo).
 
 Create a temporary clone of the repo, e.g. to `/tmp`. Then create the `tag` and
-push to origin. For example, to create a release called `v0.24.0`, you would run
-the following commands:
+push to origin. Despite Knative 1.0, the release tags in this repo are still versioned in the `v0.x.y` format and correspond to the Knative tags `knative-v1.x.y`.
+This starts from `v0.27.0` == `knative-v1.0.0` and each MINOR version addition in the `knative-v1.x.y` tag corresponds to the same addition on the `0.x.z` tag.
+
+PATCH versions are ignored in the branch name. In case a release version MUST be patched, just redo the release in the same release branch pointing to the correct commit and with the correct tag, i.e:
+
+| Branch      | Normal Tag  | Knative Tag      |
+|-------------|-------------|------------------|
+| release-1.1 | `v0.27.0`   | `knative-v1.0.x` |
```

Comment:
Sorry for bugging you (seems you missed that comment earlier), but according to your description above, this should be:

```suggestion
| release-1.0 | `v0.27.0`   | `knative-v1.0.x` |
```

Reply:
You are absolutely right 👍🏼 my b